.gitignore

```diff
@@ -8,3 +9,19 @@
 /shard.lock
 
 history.log
+
+*.dwarf
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
let's remove those empty lines and then we're good to merge!

Reply:
Whoops! No idea how I managed that 😕

Sorted.